packages/web-components/src/components/masthead/masthead-contact.ts

```diff
@@ -45,6 +47,18 @@ class DDSMastheadContact extends DDSMastheadProfile {
     `;
   }
 
+  updated(changedProperties) {
+    if (changedProperties.has('expanded')) {
```

Comment:
This condition always evalutes to `true` since we're just checking for `has`, so we can remove it.

Reply:
`changedProperties` here is a `Map` of the properties that have changed during this run of the update lifecycle, so this condition is checking if `this.expanded` changed, not if `this` has `expanded`.

We want to make sure `this.expanded` has changed before we implement side effects.

https://lit.dev/docs/components/lifecycle/#updated

Reply:
Ahhh OK gotcha. I misunderstood, sorry about that. Although interestingly, if you set a breakpoint just inside the `updated` method, the first call has all the properties in the `Map` with values of `undefined`. So for the first call at least, this condition will always be `true`.

Reply:
Correct. `changedProperties` maps to the values that have changed, and each property's values correspond to their _previous_ state.

During initialization, _all_ properties change from `undefined` to whatever their initial state is. There's also a `firstUpdated` method that runs during this first setup after the `connectedCallback` method.

Reply:
Neat. Thanks for the added context Andy!